tests/e2e/tuning_config_test.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("Tuning Config", ci.NonClassicCluster, ci.FeatureTuningConfig,
 		tcService.Destroy()
 	})
 
-	It("can create/edit/delete - [id:72521]", ci.High, func() {
+	It("can create/edit/delete - [id:72521][id:72955]", ci.High, func() {
```

Comment:
Two test case ID here? Maybe we can just merge them into one case?

Reply:
updated